frontend/src/components/intake/CourtInformationForm.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useRef } from "react";
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  Heading,
+  Select,
+  Input,
+} from "@chakra-ui/react";
+
+export type CourtDetails = {
+  currentCourtStatus: string;
+  firstNationHeritage: string;
+  firstNationBand: string;
+  orderReferral: File;
```

Comment:
```suggestion
  orderReferral: File | null;
```

let's make this either type `File` or `null` so we don't need to use `null!` in the intake page